Lab2_Code.m

```diff
@@ -0,0 +1,204 @@
+%% Lab 2
+
+%% Problem 1
+% For this section, I selected my average cosmic-ray background value as 
```

Comment:
I believe Kent mentioned that the lab reports should be readable without the lab instructions available. Adding a bit about what this section is about would be good.